doc/man3/SSL_new_listener.pod

```diff
@@ -163,6 +163,13 @@ Calling SSL_accept_connection() is an error and will return NULL. One or more
 outgoing connections under a listener can then be created using the call
 SSL_new_from_listener().
 
+To disable client address validation on a listener SSL object, the flag
+B<SSL_LISTENER_FLAG_NO_VALIDATE> may be passed in the flags field of both
+SSL_new_listener() and SSL_new_listener_from().  Note that this flag only
+impacts the sending of retry frames for server address validation.  Tokens may
+still be communicated from the server via NEW_TOKEN frames, which will still
+be validated on receipt in future connections.
```

Comment:
It's my current understanding, which might be wrong, that the retry token is what we currently use to prevent an amplification attack, and so setting this option enables the amplification attack.

Reply:
You are correct in that retry tokens allow for address validation, but section 8.1 of RFC 9000 also says:
```
Connection establishment implicitly provides address validation for both endpoints
```
So if a connection is established, then the client is considered validated, and the amplification limit doesn't apply.  Disabling address validation at this point has the effect of only preventing half open/syn flood attacks (similar to the mitigation that tcp syncookies provides).  In my view, we're ok here, in that if a user wants to risk exposing themselves to an attack in which numerous clients send Initial packets and never respond to anything else.

I'm not sure where that leaves us though.  I think this is a very useful feature, and is safe in the proper environment (closed networks, where clients are trusted, or test environments).  Is this a further documentation issue?  i.e. the pod file should contain a warning that disabling this mechanism allows for unbounded connection establishment from potentially hostile sources?

Reply:
Yes, I'm mostly interesting in warnings that it's a bad idea to enable this.

Reply:
updated docs to give a more stern warning about dangers of disabling server address validation